packages/jwt/src/JWTEIP191.ts

```diff
@@ -48,7 +48,7 @@ export class JWT extends JwtBase implements IJWT {
    */
   async sign(
     payload: string | JwtPayload,
-    { issuer, subject, noTimestamp }: JwtSignOptions = {}
+    { issuer, subject, noTimestamp, expirationTimestamp }: JwtSignOptions = {}
```

Comment:
There is also standard prop of JWT `expiresOn`. Could we use it?

Reply:
Do you mean `exp`? I think `expirationTimestamp` is more clear

Reply:
Yes, the standard name is `exp`. What I meant is property of `jsonwebtoken` https://www.npmjs.com/package/jsonwebtoken#jwtsignpayload-secretorprivatekey-options-callback. It should also validate expiration

Reply:
I see `expiresIn` but it expect different value. `jsonwbetoken` want a seconds after which token is expired, but I expect a timestamp after which token is expired. Can be misleading for some peoples if we agree on the same name convention